### PR TITLE
docs(sem): add secret protection step in different docs

### PR DESCRIPTION
**Pull request by nerda-codes**
### Your checklist for this pull request

- Added the secret protection step in the secret/folder creation docs + quickstart

### CODE REVIEW

File: identity-and-access-management/secret-manager/concepts.mdx [hunk 1]
```diff
@@ -8,6 +8,32 @@ content:
...
+An ephemeral policy is a statement that defines the lifetime of a secret and its versions, using either of the following options:
+
+- **Single access**: allows you to set your secret versions to **expire after one single access**.
+- **Time to Live**: allows you to set a time frame of up to one year, during which your secret versions are valid and accessible.
+
+The ephemeral policy also allows you to decide whether to disable or delete your secret's versions, once the secret has expired.
+
+<Message type="important">
+  - The ephemeral policy can only be applied to a secret at creation, and **cannot be removed** once applied.
+  - If applied on a secret, all its versions inherit from the policy.
```
Comment (RoRoJ) [inline identity-and-access-management/secret-manager/concepts.mdx:24]:
```suggestion
  - If applied to a secret, all its versions inherit from the policy.
```

**Review by RoRoJ — APPROVED** [review]

File: identity-and-access-management/secret-manager/concepts.mdx [hunk 2]
```diff
@@ -8,6 +8,32 @@ content:
...
+<Concept>
+
+## Ephemeral policy
+
+An ephemeral policy is a statement that defines the lifetime of a secret and its versions, using either of the following options:
+
+- **Single access**: allows you to set your secret versions to **expire after one single access**.
+- **Time to Live**: allows you to set a time frame of up to one year, during which your secret versions are valid and accessible.
+
+The ephemeral policy also allows you to decide whether to **disable** or **delete** your secret's versions, once the secret has expired.
```
Comment (nerda-codes) [inline identity-and-access-management/secret-manager/concepts.mdx:20]:
```suggestion
The ephemeral policy also allows you to decide whether to **disable** or **delete** your secret's versions, once they expire.
```

File: identity-and-access-management/secret-manager/concepts.mdx [hunk 3]
```diff
@@ -8,6 +8,32 @@ content:
...
+  - If applied to a secret, all its versions inherit from the policy.
+</Message>
+
+</Concept>
+
+<Concept>
+
+## Ephemeral secret
+
+An ephemeral secret is a secret that has a limited lifetime, enforced by an [ephemeral policy](#ephemeral-policy).
```
Comment (nerda-codes) [inline identity-and-access-management/secret-manager/concepts.mdx:33]:
```suggestion
An ephemeral secret is a secret whose versions have a limited lifetime, enforced by an [ephemeral policy](#ephemeral-policy).
```

File: identity-and-access-management/secret-manager/concepts.mdx [hunk 4]
```diff
@@ -8,6 +8,32 @@ content:
 tags: secret-manager secret version
 ---
 
+<Concept>
+
+## Ephemeral policy
+
+An ephemeral policy is a statement that defines the lifetime of a secret and its versions, using either of the following options:
```
Comment (nerda-codes) [inline identity-and-access-management/secret-manager/concepts.mdx:15]:
```suggestion
An ephemeral policy is a statement that defines the lifetime of a secret's versions, using either of the following options:
```

**Comment by crlptl** [comment]
I am Rataxes and I approve this PR

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**